checker.cs

```diff
@@ -1,34 +1,81 @@
 using System;
-using System.Diagnostics;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
-class Checker
+namespace Checker1
 {
-    static bool vitalsAreOk(float bpm, float spo2, float respRate) {
-        if(bpm < 70 || bpm > 150) {
-            return false;
-        } else if(spo2 < 90) {
-            return false;
-        } else if(respRate < 30 || respRate > 95) {
-            return false;
+
+    public class bpmcheck
+    {
```

Comment:
Repetition between the ...check classes is high. This leads to inconsistency, like 'less' in some places and 'low' in others. Remove the repetition - most of these functions are same except for the name and the limits. Accept them in the common class constructor.